### DISCUSSION TITLE
libopus

### DISCUSSION THREAD

**Discussion by HGStone** [Extrasklep/brstm#13] [Ideas]
this tool works great for alot of totk files, but there are some bwav files now encoded with libopus.. do you think it would be possible for you or someone else to incorporate this into this tool?

**Comment by potatophysicist** [comment]
Yes, support for bwav files encoded with libopus would be greatly appreciated! Totk cutscenes use 6-channel bwavs encoded with libopus and there doesn't seem to be any way to create those files at the moment.
